runelite-client/src/main/java/net/runelite/client/util/PvPUtil.java

```diff
@@ -55,10 +55,17 @@ public static int getWildernessLevelFrom(WorldPoint point)
 	public static boolean isAttackable(Client client, Player player)
 	{
 		int wildernessLevel = 0;
-		if (!(client.getVar(Varbits.IN_WILDERNESS) == 1 || WorldType.isPvpWorld(client.getWorldType())))
+		if (!(client.getVar(Varbits.IN_WILDERNESS) == 1
+				|| WorldType.isPvpWorld(client.getWorldType())
+				|| client.getWorldType().contains(WorldType.DEADMAN)
+		))
 		{
 			return false;
 		}
+		if (client.getWorldType().contains(WorldType.DEADMAN))
```

Comment:
This should be `WorldType.isDeadmanWorld(client.getWorldType())` to be inline with the other WorldType calls.

Reply:
Done